docs/resources/azure_streaming_function.md

```diff
@@ -0,0 +1,98 @@
+---
+title: About the azure_streaming_analytics_function Resource
+platform: azure
+---
+
+# azure_streaming_analytics_function
+
+Use the `azure_streaming_analytics_function` InSpec audit resource to test properties and configuration of an Azure streaming analytics function.
+
+## Azure REST API version, endpoint and http client parameters
+
+This resource interacts with api versions supported by the resource provider.
+The `api_version` can be defined as a resource parameter.
+If not provided, the latest version will be used.
+For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint, and default values for the http client will be used.
+For more information, refer to the resource pack [README](../../README.md).
+
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure).
+For an example `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
+## Syntax
+
+`resource_group`,`job-name/name` and `function-name`  must be given as a parameter.
+```ruby
+describe azure_streaming_analytics_function(resource_group: 'inspec-rg', job-name: 'my_app', function-name: 'my-function') do
+  it { should exist }
+end
+```
+
+## Parameters
+
+| Name                                  | Description                                                                       |
+|---------------------------------------|-----------------------------------------------------------------------------------|
+| resource_group                        | Azure resource group that the targeted resource resides in. `MyResourceGroup`     |
+| name                                  | Name of the webapp to test. `my_function`                                    |
```

Comment:
It looks like `name` is deprecated and `job-name` is the correct parameter. Is that correct?

Reply:
Updating